src/ssFixFruit.lua

```diff
@@ -0,0 +1,234 @@
+---------------------------------------------------------------------------------------------------------
```

Comment:
This file needs to be removed. We dont use it anymore.
Also remove it from the mod.lua